src/Core/src/Platform/Windows/WindowsNativeMessageIds.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Microsoft.Maui
+{
+	public static class WindowsNativeMessageIds
+	{
+		public const int WM_DPICHANGED = 0x02E0;
+		public const int WM_DISPLAYCHANGE = 0x007E;
+		public const int WM_SETTINGCHANGE = 0x001A;
+		public const int WM_THEMECHANGE = 0x031A;
+	}
```

Comment:
How do we want to do this? Just these? All? This casing? Pascal case?

Maybe for now we can keep this internal?

Reply:
Yeah we could just keep it internal for our own uses and maybe doc a url to a good way to find the id's.

I was torn between trying to provide some consts here for common events users may want, and exposing more individual events that propagate from these and get exposed in a more strongly named/typed way (like adding an `OnThemeChanged` to the lifecycle)

Reply:
Or maybe an enum like https://github.com/CommunityToolkit/Microsoft.Toolkit.Win32/blob/b9340a647db5cf011ac4056ee9805d56e8b91cd7/Microsoft.Toolkit.Win32.UI.Controls/Interop/Win32/WM.cs?